### PR TITLE
prevents autoequip to locked lockboxes

### LINKED ISSUE #3613: Able to put items into a locked lockbox when your bag is full.
If you have a bag on your back, full, and a lockbox in your hand and you press 'e' (or whatever your respective key is), the object in your hand will go into the lockbox, which is locked.

### PR DESCRIPTION
**Pull request by VampyrBytes**
As title
fixes #3613

### CODE REVIEW

File: code/game/objects/items/weapons/storage/lockbox.dm [hunk 1]
```diff
@@ -52,6 +52,13 @@
 			..()
 		return
 
+/obj/item/weapon/storage/lockbox/can_be_inserted(obj/item/W as obj, stop_messages = 0)
+	if(!(locked))
+		return ..()
+	if(!(stop_messages))
+		usr << "<span class='notice'>[src] is locked!</span>"
```
Comment (FlattestGuitar) [inline code/game/objects/items/weapons/storage/lockbox.dm]: Won't that create problems if you have another container that isn't locked?

Comment (VampyrBytes) [inline code/game/objects/items/weapons/storage/lockbox.dm]: No. This is an override for lockboxes, so only lockboxes call it, other containers still use the parent proc

Comment (FlattestGuitar) [inline code/game/objects/items/weapons/storage/lockbox.dm]: Ah, makes sense. :100: / :100:

**Comment by Gummin** [comment]
I accidentally lost the medicine that would have saved someones life in a lockbox once, it was horrible. :+1:

File: code/game/objects/items/weapons/storage/lockbox.dm [hunk 2]
```diff
@@ -52,6 +52,13 @@
 			..()
 		return
 
+/obj/item/weapon/storage/lockbox/can_be_inserted(obj/item/W as obj, stop_messages = 0)
+	if(!(locked))
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/storage/lockbox.dm]: the inner `()` braces of this make it less readable, `if(!locked)` and `if(!stop_messages)` would work fine

**Comment by Fox-McCloud** [comment]
I don't think the message is necessary, to be honest---when this occurs, it's only if you use quick_equip---which just skips over things that aren't supposed to be able to be inserted into---a message probabbbbly isn't necessary.

This also needs to be added to secure storage (the briefcases with locks).

http://pastebin.com/mFHst4Ng

I'd personally recommend that.

**Comment by VampyrBytes** [comment]
@Fox-McCloud 
I can add it to secure storage, no problem.
I disagree with the comment about the message though. A child proc should always deal with everything the parent proc deals with. You're right that at the moment it makes no difference, but someone who uses the proc in the future will just look at what the parent proc deals with and assume any children deal with the same things, rather than check every child

**Pull request merged**